_layouts/page.html

```diff
@@ -2,16 +2,14 @@
 layout: default
 ---
 
-<article {% if page.feature-img %}class="feature-image"{% endif %}>
-  <header id="main" style="background-image: url('{{ site.baseurl }}/{{ page.feature-img }}')">
-    <h1 id="{{ page.title | default: "" | cgi_escape }}" class="title">
-        {{ page.title }}
-    </h1>
+<div class="home">
```

Comment:
Removing the first `article` tag and replacing with this would mean that all pages with `layout:page`, will have `header_text_feature_image `  and `about_header_text ` in the hero.

For example http://0.0.0.0:4000/tags/. It comes up with the header-text `About`.

Might be better to make a seperate layout for this instead of changing `page.html` and inherit that layout in `search.html` and `about.html`.